htop.1.in

```diff
@@ -62,7 +62,7 @@ Show only the given PIDs
 Sort by this column (use \-\-sort\-key help for a column list).
 This will force a list view unless you specify -t at the same time.
 .TP
-\fB\-u \-\-user=USERNAME\fR
+\fB\-u \-\-user=USERNAME|UID\fR
```

Comment:
the strictly correct way to notate this would be
```
.It Fl u , -user Ns = Ns Ar USERNAME Ns \&| Ns Ar UID
```
which, if you limit yourself to four-font output would be
```
\fB\-u\fR, \fB\-\-user\fR=\fIUSERNAME\fR|\fIUID\fR
```

You seem to have limited yourself to two fonts and lack of punctuation, so
```
\fB\-u \-\-user=USERNAME|UID\fR
```
ugly as sin though it may be, fits well with the rest of the page.
I'm not sure what you gain, aside from noise and potential confusion, from
```
\fB\-u \-\-user={USERNAME|UID}\fR
```
What would the braces group or guard against?